frontend/src/components/pages/VolunteerDashboard/ShiftsPage.tsx

```diff
@@ -0,0 +1,108 @@
+import {
+  Box,
+  Button,
+  Container,
+  Flex,
+  Stack,
+  StackDivider,
+  Text,
+  VStack,
+} from "@chakra-ui/react";
+import React, { useContext, useEffect, useState } from "react";
+import { useHistory } from "react-router-dom";
+
+import VolunteerAPIClient from "../../../APIClients/VolunteerAPIClient";
+import * as Routes from "../../../constants/Routes";
+import AuthContext from "../../../contexts/AuthContext";
+import {
+  CheckInWithShiftType,
+  ScheduleWithShiftType,
+} from "../../../types/VolunteerTypes";
+import VolunteerShiftCard from "./ShiftCard";
+
+const ScheduledVolunteerShiftsPage = () => {
+  const { authenticatedUser } = useContext(AuthContext);
```

Comment:
i think we can use authenticatedVolunteerContext and then we will not need to make the extra call to get volunteer if in useEffect